README.md

```diff
@@ -725,3 +725,60 @@ Error Connecting to auth service ...
 }
 ```
 
+
+
+
+### Get Importer Mappings
+
+This endpoint returns a list of available staging importer apps that have been whitelisted. This endpoint also
+returns a mapping between a list of files and predicted importer app.
+
+For example,
+ * if we pass in nothing we get a response with a list of apps, and no mappings
+ * if we pass in a list of files, such as ["file1.fasta", "file2.fq", "None"], we would get back a response
+ that maps to Fasta Importers and FastQ Importers, with a weight of 0 to 1 
+ which represents the probability that this is the correct importer for you.
+ * for files for which there is no predicted app, the return is a None/null value
```

Comment:
None is python - the response is json, so just say `null`.

Reply:
done